working-with-data/guide-import-csv/guide-import-csv.adoc

```diff
@@ -140,17 +140,20 @@ It combines multiple aspects in a single operation:
 
 ===== Memory Config
 
-* Make sure to have enough memory (at least 4G heap in `neo4j-wrapper.conf`) reserved for your Neo4j-Server or Neo4j-Shell (`export JAVA_OPTS="-Xmx4G"`)
-* Configure the memory mapping settings (Neo4j 2.1) according to your expected file sizes +
-  (Neo4j 2.1 `neo4j.properties` keep these ratios: `nodestore=100M, relationshipstore=2G, propertystore=500M, stringstore=500M`)
+* Make sure to set the heap size so a sufficient value.
+You do this by setting `dbms.memory.heap.initial_size` and `dbms.memory.heap.max_size` to at least 4G in `neo4j.conf`.
+If using cypher-shell, the memory is set by doing `export JAVA_OPTS="-Xmx4G"` before starting cypher-shell.
```

Comment:
I don't think this advice is necessary anymore. Cypher-shell does not use more memory during a csv import. This was probably a piece of advice intended for `neo4j-shell` when running in "offline" mode, where an embedded database was started. Cypher-shell does not have this functionality, it is all server-side.

The entire sentence can be removed IMO.